erpnext/support/doctype/issue/issue.js

```diff
@@ -20,13 +60,58 @@ frappe.ui.form.on("Issue", {
 				});
 			}, __("Make"));
 		} else {
+			if (frm.doc.service_level_agreement) {
+				frm.dashboard.clear_headline();
+
+				let agreement_fulfilled = (frm.doc.agreement_fulfilled == "Fulfilled") ?
+					{"indicator": "green", "msg": "Service Level Agreement has been fulfilled"} :
+					{"indicator": "red", "msg": "Service Level Agreement Failed"};
+
+				frm.dashboard.set_headline_alert(
+					'<div class="row">' +
+						'<div class="col-xs-12">' +
+							'<span class="indicator whitespace-nowrap '+ agreement_fulfilled.indicator +'"><span class="hidden-xs">'+ agreement_fulfilled.msg +'</span></span> ' +
+						'</div>' +
+					'</div>'
+				);
+			}
+
 			frm.add_custom_button(__("Reopen"), function () {
 				frm.set_value("status", "Open");
 				frm.save();
 			});
 		}
 	},
 
+	priority: function(frm) {
+		if (frm.doc.service_level_agreement) {
+			frappe.show_alert({
+				indicator: 'green',
+				message: __('Changing Priority.')
+			});
+			frm.call('change_service_level_agreement_and_priority', {
+				"priority": frm.doc.priority,
+				"service_level_agreement": frm.doc.service_level_agreement
+			}).then(() => {
+				frappe.msgprint(__("Issue Priority changed to {0}.", [frm.doc.priority]));
+				frm.refresh();
+			});
+		}
+	},
+
+	service_level_agreement: function(frm) {
+		frappe.show_alert({
+			indicator: 'green',
+			message: __('Changing Service Level Agreement.')
+		});
+		frm.call('change_service_level_agreement_and_priority', {
+			"service_level_agreement": frm.doc.service_level_agreement
+		}).then(() => {
+			frappe.msgprint(__("Service Level Agreement changed to {0}.", [frm.doc.service_level_agreement]));
```

Comment:
`show_alert` again ?

Reply:
reverted to only one `msgprint`on change